lib/model/coordinate.dart

```diff
@@ -5,21 +5,47 @@ import 'package:google_maps_flutter/google_maps_flutter.dart';
 class Coordinate {
   final double _lat;
   final double _lng;
-  final String _floor;
-  final String _building;
-  final String _campus;
+  String _floor;
+  String _building;
+  String _campus;
+
+  //attributes to be used for google places search results
+  String _gPlaceTitle;
+  //String _gPlacePhotos;
```

Comment:
At the moment, if it is uneeded, you can remove this comment

Reply:
That's true, I removed it.